### ISSUE TITLE
item need allow Function Component

### ISSUE THREAD

**Issue by xxbld** [tangbc/vue-virtual-scroll-list#137]
Invalid prop: type check failed for prop "item". Expected Object, got Function

**Comment by tangbc** [comment]
Can you paste your main code snippet ?

**Comment by johannes-z** [comment]
This happens when the `item` component is a class using e.g. `vue-property-decorator`:

```typescript
import { Vue, Component, Prop } from 'vue-property-decorator'

@Component({
  components: {},
})
export default class FilterItem extends Vue {
}
```

This works:
```javascript
export default {
  
}
```

**Comment by xxbld** [comment]
@johannes-z  
https://github.com/tangbc/vue-virtual-scroll-list/pull/142

**Comment by tangbc** [comment]
Released version 1.4.3

**Comment by Rasanjika** [comment]
I'm using item component in a class using vue-property-decorator
same error comes for me.
Any solution ?
